react/src/components/SessionList.tsx

```diff
@@ -123,28 +165,144 @@ const SessionList: React.FC<SessionListProps> = ({
           {
             title: t('session.Status'),
             dataIndex: 'status',
+            render(value, record) {
+              return (
+                <>
+                  {record.status_info !== '' ? (
+                    <DoubleTag
+                      values={[
+                        { label: value, color: _.get(statusTagColor, value) },
+                        {
+                          label: record.status_info,
+                          color: _.get(statusInfoTagColor, record.status_info),
+                        },
+                      ]}
+                    />
+                  ) : (
+                    <Tag color={_.get(statusTagColor, value)}>{value}</Tag>
+                  )}
+                </>
+              );
+            },
           },
           {
             title: t('general.Control'),
           },
           {
             title: t('session.Configuration'),
+            dataIndex: 'mounts',
+            render(value, record) {
+              return (
+                <>
+                  {value.length > 0 ? (
+                    <Flex gap="xxs" style={{ width: 200 }}>
+                      <FolderOutlined />
+                      <Typography.Text
+                        ellipsis={{
+                          tooltip: (
+                            <Flex direction="column" align="start">
+                              {_.map(value, (mountedFolder) => {
+                                return <div>{mountedFolder}</div>;
+                              })}
+                            </Flex>
+                          ),
+                        }}
+                      >
+                        {value.join(', ')}
+                      </Typography.Text>
+                    </Flex>
+                  ) : (
+                    <Flex gap="xxs" style={{ color: token.colorTextDisabled }}>
+                      <FolderOutlined />
+                      No mount
+                    </Flex>
+                  )}
+                  <Flex gap="xxs">
+                    <GroupOutlined />
+                    {record.scaling_group}
+                  </Flex>
+                  <Flex gap={'xxs'}>
```

Comment:
```suggestion
                  <Flex gap="xxs">
```
Please unify the representation of gap props. (use just string)